docker/README.md

```diff
@@ -33,8 +33,7 @@ docker-compose stop
 docker-compose rm -v
```

Comment:
Can you add a section about "Code updates", describing the work flow when making changes a) python code (do nothing); b) templates (do nothing); c) frontend code (`docker-compose exec web make css js`).